paddle/framework/attr_type.proto

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Are we going to put our xx.proto file into the dir that need to use it or just in paddle/proto?